include/swift/ABI/Task.h

```diff
@@ -188,12 +190,16 @@ class AsyncTask : public HeapObject, public Job {
   /// Reserved for the use of the task-local stack allocator.
   void *AllocatorPrivate[4];
 
+  /// The next task in the linked list of waiting tasks.
+  AsyncTask *NextWaitingTask;
```

Comment:
It's intended that `Job::SchedulerPrivate` is for things like this.

Reply:
Okay, I'm using it now.